phpmailer/phpmailer.php

```diff
@@ -46,30 +42,30 @@ function phpmailer_emailer_send_prepare(App $a, array &$b)
 	// Passing `true` enables exceptions
 	$mail = new PHPMailer(true);
 	try {
-		if (!empty($a->config['system']['smtp']) && (bool)$a->config['system']['smtp'] === true) {
+        if (Config::get('phpmailer', 'smtp')) {
```

Comment:
Code standards: We use tabulation characters for indentation project-wise, please convert the indenting spaces to 2 tabs.

Reply:
Fixed